referenceframe/frame_system.go

```diff
@@ -308,6 +314,74 @@ func (sfs *simpleFrameSystem) FrameSystemSubset(newRoot Frame) (FrameSystem, err
 	return newFS, nil
 }
 
+// FrameSystemToPCD takes in a framesystem and returns a map where all elements are
+// the point representation of their geometry type with respect to the world.
+func FrameSystemToPCD(system FrameSystem, inputs map[string][]Input, logger *zap.SugaredLogger) (map[string][]r3.Vector, error) {
```

Comment:
Eric maintains a fork of SugaredLogger for Viam's internal use that should be used here. Usage does not change, only typing.

`import "github.com/edaniels/golog"`
`logger golog.Logger`